WINDOWS.md

```diff
@@ -0,0 +1,111 @@
+
+After installation, open a new cmd or shell, and you will be able to run `go version`
+
+### 4. Downloading Make
+Make is a tool which controls the generation of executables and other non-source files of a program from the source files. It is necessary for building *`makefiles`*.
+
+Make does not come with Windows, so we need to download the make binary which you can find provided by GNU [here](https://gnuwin32.sourceforge.net/packages/make.htm) and download the Binaries zip, or go to [this link](https://gnuwin32.sourceforge.net/downlinks/make-bin-zip.php) directly and begin downloading.
+
+1. Extract the downloaded zip file
+2. Go to the *`bin`*  folder, copy *`make.exe`*
+3. Go to *`C:/Program Files/Git/mingw64/bin`* and paste *`make.exe`*
+4. Through the control panel or your start menu, search and go to **Edit the system environment variables**, on Advanced tab, click Environment Variables, click Path and choose Edit, and add `C:\Program Files\Git\mingw64\bin`
+
+![](https://i.imgur.com/dyK2YMm.png)
+
+After finishing the steps above, open a new cmd or shell, and you will be able to run `make --version`
+
+### 5. Installing GCC
+GCC is the GNU compiler collection and it is necessary for compiling [cgo](https://pkg.go.dev/cmd/cgo).
+
+GCC does not come with Windows, and the best compatible setup I've found was using [this guide](https://www.guru99.com/c-gcc-install.html) which works on both Windows Server and Windows Desktop including Win 11.
+
+1. Go to [codeblocks](http://www.codeblocks.org/downloads/binaries/) and select the Windows option, then download the `mingw-setup.exe` binary.
+2. Start installation.
+3. Keep default component selection.
+
+![](https://i.imgur.com/EeZUeJU.png)
+
+4. Choose installation folder and click Install.
+ 
+![](https://i.imgur.com/D25unIm.png)
+
+5. After installation, no need to run *Code::Blocks* and click Next then Finish.
+
+![](https://i.imgur.com/rvJmv9t.png)
+
+6. Go to your installation folder i.e. *`C:\Program Files\CodeBlocks\MinGW\bin`* and you should find **gcc.exe**
+7. Through the control panel or your start menu, search and go to **Edit the system environment variables**, on Advanced tab, click Environment Variables, click Path and choose Edit, and add `C:\Program Files\CodeBlocks\MinGW\bin` (choose the path from step 4 + \MinGW\bin)
+
+After finishing the steps above, open a new cmd or shell, and you will be able to run `gcc --version`
+
+## Running Interchain Test
+
+1. Start the docker daemon by running **Docker Desktop** on Win 10/11 or using [**Docker Quickstart Terminal**](https://github.com/microsoft/docker/blob/master/docs/installation/windows.md#using-the-docker-quickstart-terminal) if you have installed Docker Toolbox.
+2. Clone the Local Interchain Repo.
+```bash
+git clone https://github.com/Reecepbcups/local-interchain.git 
+cd local-interchain
+```
+3. Run `make install`
+4. Run `local-ic`
```

Comment:
Explain setting the CONFIG variable.

can you do CONFIG=basic.json local-ic

or do you have to always
set CONFIG=basic.json

just useful for windows people